src/VisualStudio/IntegrationTests/VisualBasic/BasicBuild.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class BasicBuild
+    {
+        private IntegrationHost _host;
+
+        public BasicBuild(IntegrationHost host)
+        {
+            _host = host;
+            _host.Initialize();
```

Comment:
I see this pattern a lot:

``` C#
private IntegrationHost _host:

public .ctor(IntegrationHost host)
{
    _host = host;
    _host.Initialize();

    // other stuff
}
```

I'd rather this be in a common abstract base class to prevent duplication and make future refactoring easier.  Also, maybe another class inheriting from the base for the ones with `_interactiveWindow` above?